packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/{{#cicd_github}}.github{{/cicd_github}}/workflows/fastlane_ios_custom_build_and_deploy.yaml

```diff
@@ -32,9 +32,9 @@ on:
         description: 'Should the app be published to the App store'
         required: false
     secrets:
-      mobile_distribution_encryption_password:
+      credential_encryption_password:
```

Comment:
Do you think that we need to record that portion of the video again ?

Reply:
I was thinking of that ever since I saw the comment and there are two points from which I am looking at it:
1. The video tutorial we currently have is not yet publicly available so it could introduce confusion with the new version. So re-recoding the part makes sense if we are not going to change this too much in the future. Depending on when we're deciding to release the video tutorial(s), personally I would leave it until then.
2. Since older versions of the `rx_bloc_cli` exist, if someone would decide to generate a project, they would miss out on the setup part which has changed. Since we are also recording a new guide (for codemagic), we can include those changes there and/or briefly mention that the approach has changed to a new one.

But with that in mind, I am open to whichever approach we decide on. What I think is important is to decide and stick to an approach that won't change too much over time, since changes are required eventually. And of course communicate that clearly.

Reply:
I believe the video recording should correspond to the latest version of the rx_bloc_cli. Let's create a ticket for it and plan it accordingly. I understand that this require some effort but I really believe that if you master the video editing this will help a lot since you need to record a couple of seconds (where you show this variable) instead of the entire section of the video recording. The other option I think of is to do it together so that you know what actually to record, which will reduce the effort

Reply:
I've created a ticket for this kind discussion [here](https://github.com/Prime-Holding/rx_bloc/issues/736).